190114_tema/src/ro/sci/teo/ian14/Main.java

```diff
@@ -0,0 +1,110 @@
+package ro.sci.teo.ian14;
+
+import java.util.*;
+
+/**
+ * This is the main class, where all the program parts are put together.
+ *
+ * @author Teo
+ */
+public class Main {
+    public static void main(String[] args) {
+        //QUESTIONS:
+        //How do i make the product´s id unique? I considered it unique by default.
```

Comment:
you could generate the id at creation. Either a UUID or have a static counter:
private static int counter = 0;
and in the constructor do:
this.id = ++counter;